ui/next.config.js

```diff
@@ -1,4 +1,18 @@
 /** @type {import('next').NextConfig} */
+const CSPDirectives = [
+    "default-src 'self';",
+    "script-src 'self' 'unsafe-eval' 'unsafe-inline';",
+    "style-src 'self' 'unsafe-inline';",
```

Comment:
As above, unsafe-inline shouldn't be used.

Reply:
Though I recall having issues with tailwind and needing unsafe-inline (not sure what CSS you're using)

Reply:
The styles weren't loading unless I used unsafe-inline. We are using tailwind, so maybe that's why

Reply:
:+1: That's fair enough, annoying issue with tailwind. unsafe-inline on styles isn't nearly as bad as scripts. Necessary evil for now.